sonic_sfp/sff8436.py

```diff
@@ -364,6 +364,41 @@ class sff8436InterfaceId(sffbase):
                  'type'  : 'bitmap',
                  'decode': {}}}
 
+    sfp_type = {
+        'type':
+            {'offset':0,
+             'size':1,
+             'type' : 'enum',
```

Comment:
Delete space before `:` for consistent PEP8 style (`{''type': 'enum',`).

There are many more occurrences. Please do this throughout the entire file.